src/components/Navigation/ConversationsList/Conversation.vue

```diff
@@ -160,6 +160,17 @@ export default {
 				return 'icon-close'
 			}
 			return 'icon-delete'
+		},
+		shortMessageForSubtitle() {
```

Comment:
Tested and works :) 
I'm not sure about the name though.. Can we come up with something more descriptive? Also, could u put some JSDocs with a short description of what it does?

Reply:
Improved it a bit